src/integrationtest/java/labapi/LabService.java

```diff
@@ -16,6 +18,17 @@
 
     static ConfidentialClientApplication labApp;
 
+    static ObjectMapper mapper = new ObjectMapper()
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+
+    static <T> T convertJsonToObject(final String json, final Class<T> clazz) {
```

Comment:
Are you able to use JsonHelper here? Would avoid having duplicate code

Reply:
will have to make it public